src/commands/indexer/index.ts

```diff
@@ -1345,10 +1355,12 @@ export default class Indexer extends HealthCheck {
     input.updateNftInput.tx = transaction.hash
 
     try {
-      const response = await this.apiService.sendMutationRequest(mutation, input)
+      const structuredLogInfo = {network: network, tagId: tags}
+      const {data: response, headers} = await this.apiService.sendMutationRequest(mutation, input, structuredLogInfo)
+      const requestId = headers.get('x-request-id')
       this.networkMonitor.structuredLog(
         network,
-        `Successfully updated NFT with transaction hash ${response.updateNft?.tx}`,
+        `Successfully updated NFT with transaction hash ${response.updateNft?.tx} with requestId = ${requestId}`,
```

Comment:
@alexanderattar @ACC01ADE @BravoNatalie So i initially wrote the ticket to give us access to the `requestId` field from the API as fast as possible. Before we move forward do you think we should add requestId as an optional `structuredLogInfo` field? So maybe we can see logs like this.

```
[2022-12-21T21:48:26.233Z] [Operator] [Matic] [37113569] Block retrieved
[2022-12-21T21:48:26.486Z] [Operator] [Matic] [37113569] [79c2aaa2] Checking for job hash
[2022-12-21T21:48:26.526Z] [Operator] [Matic] [37113569] [79c2aaa2] [${requestId}] Some Request response body
```

If its too much of a change for now, we can make another ticket for later. Thoughts?

Reply:
This seems like it could be helpful but would have to be optional since not all logs hit the API

Reply:
This can already be done by using the tags variable which is last option of that function. Anything added to tags will add an extra square bracket in the log outputs. Tags can be single string, number, or array of them.

Reply:
oh yeah that's right! 👍